app/connectors/httpParsers/AppealSubmissionHTTPParser.scala

```diff
@@ -40,6 +40,10 @@ object AppealSubmissionHTTPParser {
               PagerDutyHelper.log("AppealSubmissionReads", INVALID_JSON_RECEIVED_FROM_PENALTIES)
               Left(InvalidJson)
           }
+        case CONFLICT =>
+          PagerDutyHelper.log("AppealSubmissionReads", DUPLICATE_APPEAL)
```

Comment:
We don't want to log a PD when a duplicate appeal is submitted, its a valid case